testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/CustomButtonViewWithArrow.kt

```diff
@@ -33,32 +33,42 @@ class CustomButtonViewWithArrow @JvmOverloads constructor(
 
         context.theme.obtainStyledAttributes(attrs, R.styleable.CustomButtonViewWithArrow, 0, 0)
             .let {
-                siteSection = it.getString(R.styleable.RatCustomAttributes_siteSection) ?: ""
-                pageName = it.getString(R.styleable.RatCustomAttributes_pageName) ?: ""
-                val index = it.getInt(R.styleable.RatCustomAttributes_actionType, 0)
-                if (index > -1) action = ActionType.values()[index]
                 btnLabel = it.getString(R.styleable.CustomButtonViewWithArrow_titleLabel) ?: ""
                 btnDetails = it.getString(R.styleable.CustomButtonViewWithArrow_titleDetails) ?: ""
                 if(btnDetails == "") tv_details.visibility = View.GONE else tv_details.visibility = View.VISIBLE
                 val isArrowEnable =
                     it.getBoolean(R.styleable.CustomButtonViewWithArrow_rightArrowEnable, true)
+
+                val iconRes = it.getDrawable(R.styleable.CustomButtonViewWithArrow_icon)
+
                 tv_label.text = btnLabel
                 tv_details.text = btnDetails
+
                 if (isArrowEnable) img_arrow_right.visibility =
                     View.VISIBLE else img_arrow_right.visibility = View.INVISIBLE
 
-                DemoAppAnalytics.init(AppSettings.instance.projectIdForAnalytics).sendAnalytics(
-                    RATEvent(
-                        event = EventType.APPEAR,
-                        action = action,
-                        pageName = pageName,
-                        siteSection = siteSection,
-                        componentName = btnLabel,
-                        elementType = "ButtonWithTextArrow"
-                    )
-                )
```

Comment:
Any reason why we are removing this @khairul-alam-licon ?

Reply:
I have moved this part outside of `context.theme.obtainStyledAttributes(attrs, R.styleable.RatCustomAttributes, 0, 0)
            .let {...}` as expected. We have done the same in other RAT view component e.g. `RATButton`. 
I've synced on this with @munir-rakuten also.

Reply:
It has been moved to [CustomButtonViewWithArrow.kt#L62~L71](https://github.com/rakutentech/android-miniapp/blob/7ed6d33826b8d359fc008b1beff371e8849865a5/testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/CustomButtonViewWithArrow.kt#L62~L71) now.